### PR TITLE
Adding ECR Helper for AL2. Ubuntu was not found

### PR DESCRIPTION
**Pull request by JohnPreston**
ECR Helper: https://github.com/awslabs/amazon-ecr-credential-helper

*Description of changes:*
Adding ECR Helper for AL2. package for Ubuntu was not found (standard/4.0)

Using this bash command allows to easily add the config

```
echo $(cat $HOME/.docker/config.json | jq '. |= . + {"credsStore": "ecr-login"}') > $HOME/.docker/config.json
```
Did not add that command to the Dockerfile as it is up to the users to decide whether they would like to use it.

### CODE REVIEW

**Comment by larstobi** [comment]
@subinataws Could I ask you to take a look at this?

File: al2/x86_64/standard/3.0/Dockerfile [hunk 1]
```diff
@@ -25,6 +25,7 @@ RUN set -ex \
     && rpm --import https://download.mono-project.com/repo/xamarin.gpg \
     && curl https://download.mono-project.com/repo/centos7-stable.repo | tee /etc/yum.repos.d/mono-centos7-stable.repo \
     && amazon-linux-extras enable corretto8 \
+    && amazon-linux-extras enable docker \
```
Comment (leoherran-aws) [inline al2/x86_64/standard/3.0/Dockerfile:28]: Why is this needed?

Comment (samuelkarp) [reply al2/x86_64/standard/3.0/Dockerfile:28]: The credential helper is available as part of the `docker` extra.  It's not included in the core yum repository.

Comment (leoherran-aws) [reply al2/x86_64/standard/3.0/Dockerfile:28]: Thanks Samuel!

**Review by leoherran-aws — APPROVED** [review]

**Comment by leoherran-aws** [comment]
Merged in 2bb700699c481aa4500ab49165b23516519693d7. Thanks for your contribution!

**Comment by fagiani** [comment]
@leoherran-aws thanks for the update.

Is there a way to use it as of now or we must wait until the next release?

Keep Rocking!

**Comment by leoherran-aws** [comment]
New AL2:3.0 image with this change is live: https://github.com/aws/aws-codebuild-docker-images/releases/tag/21.08.20
